packages/verticaltabs/src/react/__stories__/index.story.js

```diff
@@ -0,0 +1,47 @@
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+
+import VerticalTabs from '../index.js'
+
+storiesOf('Vetical Tabs', module)
+  .addDecorator(story => <div style={{ height: '100vh' }}>{story()}</div>)
+  .add('Group', _ => (
+    <VerticalTabs>
```

Comment:
in this example both the Group header and Tier2 header are `h2`. we need to device of a way where that doesn't happen - maybe it's configurable and prop to pass in the headerTag.

another option is more compound components :) - i could see something like this being more flexible

```
<VerticalTabs.Group>
  <VerticalTabs.Group.Header tag="h2">Tab Group</VerticalTabs.Group.Header>

  <VerticalTabs.List>
    <VerticalTabs.List.Header tag="h3">Tier 1 a</VerticalTabs.List.Header>

    <VerticalTabs.List.Item>Tier 2 a</VerticalTabs.List.Header>
    <VerticalTabs.List.Item>Tier 2 b</VerticalTabs.List.Header>

  </VerticalTabs.List>
```
another option is passing element as props

```
<VerticalTabs.Group heading={<VerticalTabs.Header tag="h2">Tab Group</VerticalTabs.Header>}>
...
</VerticalTabs.Group>
```

Reply:
another concern is the the group heading needs to be able to have a data attrs on it for gitprime to use their onboarding tool. a more explicit compound component would add that flexibility. let me know if you want to talk about different approaches